expertino/clips/orders/order-selection.clp

```diff
@@ -62,19 +62,32 @@
   ?set-f <- (pddl-set-goals (instance ?instance) (state DONE))
   (pddl-manager (node ?node))
   (pddl-instance (name ?instance) (busy-with FALSE) (state LOADED))
+  (planning-filter (action-names $?an))
```

Comment:
do we always just want to apply one random planning filter? I think this works for now, but maybe we want to improve this (e.g., we want different planning filters for different purposes maybe).

Reply:
For now the planning filter is completely stupid, I would leave it up to people actually using it to extend the deftemplate to fit our needs, e.g., adding some ids, references to goals/plans or similar.

Reply:
just a bit dangerous because technically you could have multiple facts. But let's leave it like this for now.